niworkflows/func/util.py

```diff
@@ -316,10 +317,15 @@ def init_enhance_and_skullstrip_bold_wf(
             name='norm',
             n_procs=omp_nthreads)
         norm.inputs.fixed_image = str(bold_template)
-        map_brainmask = pe.Node(
-            ApplyTransforms(interpolation='MultiLabel', float=True, input_image=str(brain_mask)),
+        map_brainmask = pe.Node(ApplyTransforms(
+            interpolation='BSpline',
+            float=True,
+            # Use the higher resolution and probseg for numerical stability in rounding
+            input_image=str(get_template(
+                'MNI152NLin2009cAsym', resolution=1, desc='brain', suffix='probseg'))),
             name='map_brainmask'
         )
+        binarize_mask = pe.Node(Binarize(thresh_low=0.75), name='binarize_mask')
```

Comment:
I would say the default interpretation of binarizing probability is to threshold on >50%.

```suggestion
        binarize_mask = pe.Node(Binarize(thresh_low=0.5), name='binarize_mask')
```

Is there some extenuating situation here?